prometheus_client/exposition.py

```diff
@@ -38,6 +39,13 @@
 PYTHON376_OR_NEWER = sys.version_info > (3, 7, 5)
 
 
+def _get_disable_compression() -> bool:
```

Comment:
Instead of an environment variable what would you think of passing an additional argument to `make_(w|a)sgi_app`? My thought is that most users should have control over creating the apps so an env var is not necessary. We try to only use env vars in places where users might not have access to the code, such as registering metrics.

Reply:
Yeah, makes sense. I suppose we need to expose this option in `start_http_server` as well?

Reply:
I think that would be nice, though not completely required as `start_http_server` doesn't need to support all configuration for advanced user.